src/core/getEngines.ts

```diff
@@ -1,39 +1,27 @@
 import { readJsonFile } from '../utils';
-import {
-  EnginesDataArray,
-  Manager,
-  PackageJson,
-  PackageLock,
-  PackageManagerName,
-} from '../types';
+import { EnginesDataArray, Manager, PackageJson, PackageLock, PackageManagerName } from '../types';
 
 export async function getEngines(deps: Array<string>, manager: Manager): Promise<EnginesDataArray> {
-  try {
```

Comment:
i think we should probably keep this `try/catch` because the `Promise.all` in `getYarnEngines` could throw, which _I believe_ would cause an `Unhandled Promise Rejection` without it.

there's a `try/catch` in `readJsonFile` -- if it throws an unexpected error, it would cause `Promise.all` to reject and it isn't being caught at the call site. I don't think rejections like this (Promises across functions) bubble up to the next highest `catch` block, but I'm not positive.

or, alternatively, we could put a `try/catch` around the code in `getYarnEngines`.